tokenserver/assignment/memorynode.py

```diff
@@ -39,6 +39,14 @@ def get_user(self, service, email):
         except KeyError:
             return None
 
+    def should_allocate_to_spanner(self, email):
```

Comment:
This is no longer used in the memory backend, and can be removed. (Alternately, if you want to keep it in the public interface, we could consider moving it into some sort of shared helper. I don't feel particularly strongly about it, just don't like seeing unused code).

Reply:
nah, not needed.